src/clang.rs

```diff
@@ -833,6 +833,10 @@ impl Type {
     /// Get the number of template arguments this type has, or `None` if it is
     /// not some kind of template.
     pub fn num_template_args(&self) -> Option<u32> {
+        if !clang_Type_getNumTemplateArguments::is_loaded() {
```

Comment:
Can you comment on this, saying that it's not correct but that that shouldn't block C bindings from working?